docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -83,7 +83,140 @@ are:
+The following table summarizes the changes that SCU supports.  Please consult
+the detailed documentation below for additional information.
+
++-----------------+---------------+------------+-----------------------------------+
+| **Daml Model    | **Scope**     | **Covered  | **Comments**                      |
+| Change**        |               | by SCU?**  |                                   |
++=================+===============+============+===================================+
+| Rename of       | Template      | No         |                                   |
+| Module          | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Rename of       | Template      | No         |                                   |
+| Template        | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field Type      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field           | Definition    |            |                                   |
+| Ordering        |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         |                                   |
+| Field Name      | Definition    |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | A new template version needs to   |
+| Signatories     | Definition    |            | return the same signatory result  |
+|                 |               |            | for a persisted contract,         |
+|                 |               |            | otherwise the contracts cannot be |
+|                 |               |            | used by an upgraded package (this |
+|                 |               |            | fails at runtime).                |
+|                 |               |            |                                   |
+|                 |               |            | One can change the function that  |
+|                 |               |            | computes the signatories (for     |
+|                 |               |            | example to take into account new  |
+|                 |               |            | fields) when creating contracts.  |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Template      | No         | The type of a key cannot be       |
```

Comment:
This is awkward and needs to be rephrased - perhaps something like "The key's type cannot be changed - for example, you cannot add a new field to a record that the key contains."

Reply:
Change